setup.py

```diff
@@ -66,6 +66,7 @@ def git_version(gitdir, default="0.0.0"):
 
 setup(
     name="cirrus-geo",
+    python_requires=">=3.8,<=3.9",
```

Comment:
Is there a good reason to set an upper limit here?

For one, once 3.10 is available as a runtime, it seems like also forcing a cirrus-geo upgrade to make use of it despite the code being compatible is not great. For another, you can technically run the cirrus cli using a more recent python version; serverless packaging the lambdas will use whatever version of python is specified by the runtime parameter in the cirrus.yml, which can be a different version (though that is not recommended).

Because of those points I'd suggest not placing an upper bound, but I am open to opinions.

Reply:
I had originially put it at <=3.9 b/c we haven't system tested it with 3.10, but then I realized that doesn't work for restricting it to any 3.9.x version, and it should be `<3.10` instead -- but I also don't thing the restriction matters that much in practice, so I removed it.